packages/react-hot-loader/test/AppContainer.dev.test.js

```diff
@@ -968,7 +968,7 @@ describe(`AppContainer (dev)`, () => {
       )
       expect(wrapper.find('App').length).toBe(1)
       expect(wrapper.contains(<div>hey</div>)).toBe(true)
-      expect(spy).toHaveBeenCalledTimes(1)
+      expect(spy).toHaveBeenCalledTimes(2)
```

Comment:
As noted, we end up calling functional components twice now. We could cache the initial result if you want, though.